CHANGELOG.md

```diff
@@ -40,6 +40,7 @@ release.
 - Pinned cspice version to 67 [#5083](https://github.com/USGS-Astrogeology/ISIS3/issues/5083) 
 - Changed the `rsync` related commands in the ISIS SPICE Web Service document to `downloadIsisData` command
 - Updated Geos from version 3.7 to 3.9 [#3627](https://github.com/DOI-USGS/ISIS3/issues/3627)
+- Updated Ale to version 0.9.1
```

Comment:
needs a PR number (assuming we normally we put version ticks in here).

Reply:
Added